src/main/java/com/banking/application/services/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package com.banking.application.services.impl;
+
+import com.banking.application.entity.User;
+import com.banking.application.repository.UserRepository;
+import com.banking.application.services.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class UserServiceImpl implements UserService {
+    private final UserRepository userRepository;
+
+    @Override
+    public User save(User user) {
+        return userRepository.save(user);
+    }
+
+    @Override
+    public User update(User user) {
+        return userRepository.save(user);
+    }
```

Comment:
checklist
`You can have a single save() method on the service layer for both ‘create’ and ‘update’ operations.
`